auth-server-open-api-spec.yaml

```diff
@@ -577,7 +577,9 @@ components:
           description: '[ISO8601 repeating interval](https://en.wikipedia.org/wiki/ISO_8601#Repeating_intervals)'
       required:
         - interval
-        - oneOf - sendAmount - receiveAmount
+        - oneOf
+        - sendAmount
+        - receiveAmount
```

Comment:
What do we think of:
```
anyOf:
  - required:
    - interval
    - sendAmount
  - required:
    - interval
    - receiveAmount
```
https://stackoverflow.com/questions/41172240/json-schema-how-to-make-anyof-two-or-more-properties-required

Reply:
:point_up_2: that didn't work.

:point_down: this should:
- https://github.com/interledger/open-payments/pull/180

Reply:
I don't know the correct spec, I was just trying to get syntax work so yours is probably the right one.